README.md

```diff
@@ -61,6 +61,7 @@ A collection of all things awesome related to the [Dagster](https://dagster.io)
 - [Datadex](https://github.com/datonic/datadex)
 - [Anomstack](https://github.com/andrewm4894/anomstack)
 - [dagster-nomad Nomad launcher/executor for Dagster](https://github.com/paylead/dagster-nomad)
+- [Dagster Demo](https://github.com/kahnwong/dagster-demo)
```

Comment:
Very cool, @kahnwong - would you be able to add a quick highlight of the features here to differentiate it from other "Dagster Demos" that might be added in the future? For example: "Dagster Demo: Medallion Architecture, Partitioning, Auto-materializations, etc" or something along those lines. Thanks!

Reply:
Good catch, wasn't sure since I don't see description on other projects.

https://github.com/dagster-io/awesome-dagster/pull/14/commits/bb3311d4846cd4f8f80836eba7e88550c7e048eb

Please let me know if the description needs more work.

Reply:
I think that looks great, thank you @kahnwong !

Reply:
As for other examples not having descriptions, I think it's a good practice to include them. We'll have to retroactively add them to the existing links. 👍